src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -209,7 +209,7 @@ public function dump(array $options = [])
 
             if (3 <= $i) {
                 $regex = '';
-                $lastOptionalDir = $i > 8 ? $i - 5 : 3;
+                $lastOptionalDir = $i > 8 ? $i - 5 : $i - 3;
```

Comment:
3 is `var`, `cache`, `env`
I don't get why the projet dir + parent dir are optionnal when the path contains more than 8 directories